ontolib-core/src/test/java/com/github/phenomics/ontolib/ontology/data/ImmutableOntologyTest.java

```diff
@@ -36,4 +37,26 @@ public void test() {
         ontology.getParentTermIds(id1).toString());
   }
 
+
+  /** The example graph has id1->id2, id1->id3, id1->id4, id2->id5, id4->id5 */
+  @Test
+  public void testPathExists() {
+    assertTrue(ontology.existsPath(id1,id2));
```

Comment:
Please make whitespace consistent, your IDE should offer an option to do this automatically.